docs/resources/instance_server.md

```diff
@@ -163,6 +163,20 @@ resource "scaleway_instance_server" "from_snapshot" {
 }
 ```
 
+#### Using Scaleway Block Storage (SBS) volume
+
+```terraform
+resource "scaleway_instance_server" "server" {
+  type = "PLAY2-MICRO"
+  image = "ubuntu_jammy"
+  root_volume {
```

Comment:
How is it possible to find this volume from a data source? Does it have a name/id ?

Reply:
This creates a new volume, why would we use a data source ?